### ISSUE TITLE
MD Scroll misbehave on opened expansion panels

### ISSUE THREAD

**Issue by alibenmessaoud** [angular/material2#14781]
#### What is the expected behavior?

When scrolling on a list of expansion panels  with the first one opened and then returning back to the top, the first panel should still be opened. 

#### What is the current behavior?

Scrolling will open others expansion panels that have the same position as the first one, also it misbehaves the UI.

#### What are the steps to reproduce?

Just scrolling here after opening the first expansion panel: https://stackblitz.com/edit/angular-v9kaif

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angualr 7, MD 7,  TS 3

#### Is there anything else we should know?
Same thing if you choose to open the nth expansion panel on the list.

Authors: @hzitoun @alibenmessaoud

**Comment by alibenmessaoud** [comment]
Jawbou ya zebi

**Comment by crisbeto** [comment]
I don't think we can do much about this. As far as the expansion panels are concerned, they've been destroyed and re-created.
